patch/deckpatch.go

```diff
@@ -170,15 +170,11 @@ func (patch *DeckPatch) ApplyToNodes(yamlData *yaml.Node) (err error) {
 
 	// query the yamlData using the selector
 	nodes := make([]*yaml.Node, 0)
-	for _, selector := range patch.Selectors {
-		moreNodes, err := selector.Find(yamlData)
```

Comment:
Verified that it is safe to do this - selector.Find never returned errors in the old library https://github.com/vmware-labs/yaml-jsonpath/blob/v0.3.2/pkg/yamlpath/path.go#L24